org.hnefatafl.hnefatafl_client.yml

```
@@ -0,0 +1,47 @@
+app-id: org.hnefatafl.hnefatafl_client
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: hnefatafl-client
+sdk-extensions: [
+  "org.freedesktop.Sdk.Extension.rust-stable",
+  "org.freedesktop.Sdk.Extension.llvm18"
+]
```

```suggestion
sdk-extensions:
  - org.freedesktop.Sdk.Extension.rust-stable
  - org.freedesktop.Sdk.Extension.llvm18
```

if you want to use the JSON syntax make the manifest a JSON.